Dockerfile-1.7

```diff
@@ -26,19 +26,15 @@ RUN mkdir /release && \
     chown $USER /release
 
 # create directory to get source from
-RUN mkdir /src && \
-    chown $USER /src && \
-    mkdir -p /go/src/github.com/decred/dcrd && \
```

Comment:
Why did you remove these?  The `run_tests.sh` script requires them (was done that way to make it work on travis more seemlessly).